rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -331,23 +332,27 @@ public String unlockNodes(String sessionId, Set<String> urls) throws RestServerE
 
     @Override
     public String removeNode(String sessionId, String url, boolean force) throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.removeNode(sessionId, url, force));
+        return executeFunctionReturnStreamAsStringWithoutNewLines(restClient -> restClient.removeNode(sessionId,
+                                                                                                      url,
+                                                                                                      force));
     }
 
     @Override
     public String removeNodesource(String sessionId, String name, boolean preempt)
             throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.removeNodesource(sessionId, name, preempt));
+        return executeFunctionReturnStreamAsStringWithoutNewLines(restClient -> restClient.removeNodesource(sessionId,
+                                                                                                            name,
+                                                                                                            preempt));
     }
 
     @Override
     public String releaseNode(String sessionId, String url) throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.releaseNode(sessionId, url));
+        return executeFunctionReturnStreamAsStringWithoutNewLines(restClient -> restClient.releaseNode(sessionId, url));
     }
 
     @Override
     public String getVersion() throws RestServerException, ServiceException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)